color/command.c

```diff
@@ -1023,7 +1023,8 @@ static enum CommandResult color_dump(struct Buffer *buf, struct Buffer *s,
   struct PagerData pdata = { 0 };
   struct PagerView pview = { &pdata };
 
-  pdata.fname = tempfile;
+  pdata.fname = buf_strdup(tempfile);
+  buf_pool_release(&tempfile);
```

Comment:
This is an odd bit of code...
We've created a couple of temporary structs (`pdata`, `pview`) to pass to `mutt_do_pager()`.
They don't want to take ownership of `tempfile`, they just share it: `buf_string()`.
That means we can't `buf_pool_release()` until after `mutt_do_pager()`.

Reply:
good catch, thanks! Should be fixed now.